src/components/Popups/ReplyField/__tests__/ReplyField-test.tsx

```diff
@@ -54,18 +77,70 @@ describe('components/Popups/ReplyField', () => {
         });
     });
 
+    describe('getVirtualElement()', () => {
+        test('should return null if no selection', () => {
+            const wrapper = getWrapper();
+            const instance = wrapper.instance();
+
+            window.getSelection = () => null;
```

Comment:
Should this be a `jest.spyOn` call that we setup and reuse in `beforeEach`, instead?

Reply:
> Should this be a `jest.spyOn` call that we setup and reuse in `beforeEach`, instead?

I tried `jest.spyOn(window, 'getSelection')` but it throws me error `Cannot spy the getSelection property because it is not a function; undefined given instead`.

Reply:
It may not be available in jsdom, which is the environment we use for our tests. We're going to need that API for text highlights at some point, so it may be worth mocking globally.

Reply:
got it